docs/storefront/graphql/orders.mdx

```diff
@@ -316,6 +316,111 @@ In this example, the query is set up to get details for order #106. Use `entityI
   </Tab>
 </Tabs>
 
+### Get an order metafield
+In this example, the query is set up to get a metafield for order #1234000006. Use `entityId` to query by order.
+
+<Tabs items={[`Request`, `Response`]}>
+  <Tab>
+   ```graphql filename="Example query: Get order metafield" showLineNumbers copy
+    {
+        site {
+            order(filter: {entityId: 1234000006}) {
+                metafields(namespace: "namespace") {
+                    edges {
+                        node {
+                            key
+                            value
+                        }
+                    }
+                }
+            }
+        }
+    }
+```
+</Tab>
+    <Tab>
+   ``` graphQL filename="Example response: Get order metafield" showLineNumbers copy
+   {
+    "data": {
+        "site": {
+            "order": {
+                "metafields": {
+                    "edges": []
```

Comment:
Not sure there is a point showing empty response 🤔 What do you think ?

Reply:
Hi @bc-rafsierk  I agree, but I how do you populate the edges array? Do you have a better example I can use?